### PR TITLE
Animals table

### PR DESCRIPTION
**Pull request by looh16**
**Creation of the initial data structure of a veterinary clinic :**

> Vet Clinic database
> animals table
> Inserts for the animals table
> Queries in the animals table

**Screenshot with the queries results** 
List the name of all animals born between 2016 and 2019
Find all animals not named Gabumon
Find all animals that are neutered
Find all animals whose name ends in mon
Find all animals with a weight between 10 4kg and 17 3kg
List name and escape attempts of animals that weigh more than 10 5kg
List the date of birth of all animals named either Agumon or Pikachu
List the name of all animals born between 2016 and 2019
List the name of all animals that are neutered and have less than 3 escape attempts

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,25 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+--Find all animals whose name ends in "mon"
+SELECT * FROM animals WHERE name like '%mon';
+
+--List the name of all animals born between 2016 and 2019
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-3-12';
```
Comment (addod19) [inline queries.sql]: - [ ] It would be better if we change the data from `2019-3-12` to `2019-31-12` in order to make the right queries

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @looh16, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- [ ] README is descriptive :1st_place_medal: 
- [ ] Queries are supported with screenshots :100: 
- [ ] Schema table has all the required fields :rocket: 
- [ ] Commit messages are easier to understand :fireworks: 
- [ ] Git-flow followed :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (looh16) [reply queries.sql]: Yes, it makes perfect sense. Thanks!

**Review by smunozmo — APPROVED** [review]
Hi @looh16, 👋 

## Highlights 👏 

- Excellent job making the suggested changes, great! 🎉 
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

# APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by looh16** [comment]
Thanks!

**Pull request merged**